Dockerfile

```diff
@@ -4,10 +4,13 @@ ARG BASE_IMAGE_MINIMAL
 # Build node feature discovery
 FROM golang:1.17.2-buster as builder
 
-# Download the grpc_health_probe bin
+# Build and install the grpc-health-probe binary
 RUN GRPC_HEALTH_PROBE_VERSION=v0.4.6 && \
 	go install github.com/grpc-ecosystem/grpc-health-probe@${GRPC_HEALTH_PROBE_VERSION}
 
+# Rename it as it's referenced as grpc_health_probe in the deployment yamls
+RUN mv /go/bin/grpc-health-probe /go/bin/grpc_health_probe
```

Comment:
nit: could you join this to the previous `RUN` with `&&`. Just a small optimization to avoid one excess layer. WDYT?

Reply:
done